tmt/utils/markdown.py

```diff
@@ -0,0 +1,203 @@
+"""
```

Comment:
Have you checked the existing `markdown_to_html`, https://github.com/jscotka/tmt/blob/237cc0edbfeab7f2ca39347ebec4b77aa3a8104b/tmt/utils/__init__.py#L3385 ?